dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/event/RumEventDeserializerTest.kt

```diff
@@ -160,6 +161,32 @@ internal class RumEventDeserializerTest {
         )
     }
 
+    @Test
+    fun `𝕄 deserialize a serialized RUM LongTaskEvent 𝕎 deserialize()`(
+        forge: Forge
+    ) {
+        // GIVEN
+        val fakeLongTaskEvent = forge.getForgery(LongTaskEvent::class.java)
+        val fakeEvent: RumEvent = forge.getForgery(RumEvent::class.java)
+            .copy(event = fakeLongTaskEvent)
+        val serializedEvent = serializer.serialize(fakeEvent)
+
+        // WHEN
+        val deserializedEvent = testedDeserializer.deserialize(serializedEvent)
+
+        // THEN
+        assertThat(deserializedEvent).isNotNull()
+        DatadogMapAnyValueAssert.assertThat(deserializedEvent!!.globalAttributes)
```

Comment:
do you still need this `deserializedEvent!!` ?

Reply:
yes, unfortunately. Kotlin compiler is not yet that smart to infer non-null type for AssertJ :)